wpa_supplicant/ctrl_iface.c

```diff
@@ -13009,10 +13003,13 @@ static int wpas_global_ctrl_iface_fst_detach(struct wpa_global *global,
 char * wpa_supplicant_global_ctrl_iface_process(struct wpa_global *global,
 						char *buf, size_t *resp_len)
 {
-	char *reply;
-	const int reply_size = 2048;
+	static char reply[1024];
+	const int reply_size = 1024;
```

Comment:
Better
```
const int reply_size = sizeof(reply);
```
So that `reply_size` will always be correct and we do not need to sync the value if `reply` size changes.